server/lib/mappings/result-data.json

```diff
@@ -1,590 +1,189 @@
-                              },
-                              "value": {
-                                "type": "double"
-                              }
-                            }
-                          },
-                          "value": {
-                            "type": "double"
-                          }
-                        }
-                      },
-                      "server_hostname": {
-                        "type": "string"
-                      },
-                      "stddevpct": {
-                        "type": "long"
-                      }
-                    }
-                  }
-                }
-              },
-              "throughput": {
-                "properties": {
-                  "iops_sec": {
-                    "properties": {
-                      "uid": {
-                        "type": "string"
-                      },
-                      "role": {
-                        "type": "string"
-                      },
-                      "client_hostname": {
-                        "type": "string"
-                      },
-                      "mean": {
-                        "type": "double"
-                      },
-                      "description": {
-                        "type": "string"
-                      },
-                      "server_port": {
-                        "type": "string"
-                      },
-                      "closest sample": {
-                        "type": "long"
-                      },
-                      "stddev": {
-                        "type": "long"
-                      },
-                      "samples": {
-                        "properties": {
-                          "timeseries": {
-                            "properties": {
-                              "date": {
-                                "type": "long"
-                              },
-                              "read(0) or write(1)": {
-                                "type": "long"
-                              },
-                              "value": {
-                                "type": "double"
-                              }
-                            }
-                          },
-                          "value": {
-                            "type": "double"
-                          }
-                        }
-                      },
-                      "server_hostname": {
-                        "type": "string"
-                      },
-                      "stddevpct": {
-                        "type": "long"
-                      }
-                    }
-                  }
-                }
-              },
-              "parameters": {
+            "@timestamp": {
+                "type": "date",
+                "format": "dateOptionalTime"
+            },
+            "timestamp_original": {
+                "type": "long"
+            },
+            "@metadata": {
```

Comment:
We should consider making iteration summary documents, which are "parents" to sample summary documents, which are in turn "parents" to individual sample documents.  Then for each sample document just have enough metadata to identify its "coordinates" (so to speak) in the universe of data, but leaving all the detailed metadata in the summary documents.

The thinking is that folks will browse the summary documents, but will only view the actual sample graphs when they know exactly which sample(s) to view from the summaries.

Reply:
We emit a sample document, which is the parent of all the result data.  The shrinks the size of each result data document considerably.